streams/src/main/java/org/apache/kafka/streams/kstream/internals/KGroupedStreamImpl.java

```diff
@@ -320,12 +360,7 @@ public Long apply(K aggKey, V value, Long aggregate) {
                                            final StateStoreSupplier<SessionStore> storeSupplier) {
         Objects.requireNonNull(sessionWindows, "sessionWindows can't be null");
         Objects.requireNonNull(storeSupplier, "storeSupplier can't be null");
-        final Initializer<Long> initializer = new Initializer<Long>() {
-            @Override
-            public Long apply() {
-                return 0L;
-            }
-        };
+        final Initializer<Long> initializer = countInitializer;
```

Comment:
var `initializer` seems to be redundant.

Reply:
intellij refactoring playing games with me.